### ISSUE TITLE
`expo-apple-authentication` keeps loading forever

### ISSUE THREAD

**Issue by NonameLTUs** [expo/expo#11476]
Hi.

I have `expo-apple-authentication` v2.2.1 installed to project, but I can't test it on simulator. When I try to login with Apple ID, I enter password and see loading icon scrolling all the time. No logs show up in console, it just keeps loading.

<img width="412" alt="Screen Shot 2020-12-28 at 7 17 37 PM" src="https://user-images.githubusercontent.com/25826899/103231927-c44cc700-4941-11eb-8a0b-ad78a5b0b234.png">

What could be the problem?

Thanks.

**Comment by franjorub** [comment]
+1 I have the same issue keeps loading forever

**Comment by brentvatne** [comment]
this appears to just be a simulator issue, it works as expected on device. it could be a bug in the ios simulator, would need to investigate further to find out.

**Comment by brentvatne** [comment]
indeed, this appears to be a bug with the simulator: https://developer.apple.com/forums/thread/651533

**Comment by franjorub** [comment]
@brentvatne I'm worried about this, in the thread they said that the apps are rejected by this issue, do you had been this problem with apple this year ?

**Comment by brentvatne** [comment]
@franjorub - if you encounter that, then you can appeal the rejection and point the reviewer to https://developer.apple.com/forums/thread/651533

**Comment by brentvatne** [comment]
@mikehardy gave a reasonable suggestion on https://github.com/invertase/react-native-apple-authentication/issues/141#issuecomment-709364954

> In app store review comments you should note for reviewers that if they test apple sign in, it is broken on ios14 simulators and they must use either a real device or ios13

**Comment by franjorub** [comment]
I saw, thanks a lot @brentvatne

**Comment by franjorub** [comment]
@brentvatne  sorry one last question, I'm new with this, how can I change the default version coming with the emulator from Xcode ? I'm downloading the version 13.6 from components but after that can I select the version using shift + I command in the expo cli ? 
<img width="823" alt="image" src="https://user-images.githubusercontent.com/24401350/103323283-9090a080-4a18-11eb-8531-ff1e0b3d481b.png">

**Comment by brentvatne** [comment]
yeah after you download that press shift+i in expo-cli to open it

**Comment by estefi-prieto** [comment]
Hi, I'm trying to test that in a simulator with iOS 13.6 and I can see the native modal, but when I complete the password field properly it seems that success, but it doesn't give me a response. Did you have a similar problem?

**Comment by juxhinrubik** [comment]
> @brentvatne sorry one last question, I'm new with this, how can I change the default version coming with the emulator from Xcode ? I'm downloading the version 13.6 from components but after that can I select the version using shift + I command in the expo cli for example ? <img alt="image" width="823" src="https://user-images.githubusercontent.com/24401350/103323283-9090a080-4a18-11eb-8531-ff1e0b3d481b.png">

Choose your simulator version at Product>Destination>...

**Comment by dorthwein** [comment]
We are getting rejected it seems because of this - currently filing a repeal as suggested! 

https://developer.apple.com/forums/thread/651533

**Comment by PEZO19** [comment]
This issue still exists right? (Does not seem to work for me with simulator ios 17.0, but works on real device)

**Comment by tylim88** [comment]
simulator ios 17.4 still having this issue, it is driving me crazy

**Comment by frdwhite24** [comment]
I'm afraid i'm still experiencing this issue. Has anyone got any other fixes for it?

**Comment by elyobo** [comment]
Reopen, this issue remains. Apple auth works on development builds on physical devices but hangs after password in the simulator.

**Comment by bypina** [comment]
i’m getting the same issue i’m on expo 48

**Comment by dafex301** [comment]
getting same issue on expo 51.0.38

**Comment by LucaL1fe** [comment]
Anybody found a solution? I still have the issue, that it is not working on the simulator (iOS 17.4 + Expo 53)
